docs/marketplace/add_to_marketplace/contributing_overview.md

```diff
@@ -1,30 +1,29 @@
 ---
 id: contributing_overview
-title:  Contributing to the Marketplace
-sidebar_label:  Contributing to the Marketplace
+title:  Manage your Company Marketplace
+sidebar_label:  Internal Company Marketplace
```

Comment:
I am in doubt about the "Internal Company" definition: is this a clear concept explained elsewhere in our docs?

Reply:
Yes, specifically referred to the Marketplace we are defining the Private Marketplace as "Internal Company Marketplace"